internal/lockclient/simple_client_test.go

```diff
@@ -0,0 +1,72 @@
+package lockclient
+
+import (
+	"os"
+	"testing"
+
+	"github.com/GoPlayAndFun/LocKey/internal/lockservice"
+	"github.com/GoPlayAndFun/LocKey/internal/node"
+	"github.com/rs/zerolog"
+)
+
+func TestAcquire(t *testing.T) {
```

Comment:
I don't seem to wrap my head around the motive of the complexity of the test.
You don't need 2 goroutines here. One goroutine can start the server and wait until you finish the test and then the `quit` signal can be sent.

Reply:
You comments have been implemented. However, now an issue has arisen: killing the goroutine that started the server does not kill the `gracefulShutdown` goroutine.